cdap-cli/src/main/java/co/cask/cdap/cli/CLIConfig.java

```diff
@@ -127,25 +127,28 @@ public TableRenderer getTableRenderer() {
   }
 
   public Id.Namespace getCurrentNamespace() {
-    return clientConfig.getNamespace();
+    return connectionConfig != null ? connectionConfig.getNamespace() : null;
```

Comment:
Not having a ConnectionConfig means the CLI isn't connected to any cluster (you'll have the prompt `cdap (DISCONNECTED)>`). I'll change it to not allow returning null though.